contrib/packs/actions/pack_mgmt/download.py

```diff
@@ -141,10 +141,12 @@ def _eval_subtree(repo_url, subtree):
     @staticmethod
     def _eval_repo_url(repo_url):
         """Allow passing short GitHub style URLs"""
+        has_git_extension = re.compile('.git$')
         if len(repo_url.split('/')) == 2 and not "git@" in repo_url:
-            return "https://github.com/{}.git".format(repo_url)
```

Comment:
So, it's sort of all over the place.

I saw that some of the action metadata has `https` defined, while the `download.py` had `git` as the protocol defined. In my original PR, I went toward HTTPS to avoid having git keys splayed all over the place, but I kept running into issues with prompting for HTTPS logins on clean installs.

I think we should be on HTTPS, but the change broke things. I'm looking into why I'm being prompted for auth on public repos this AM, and may end up changing this back ultimately.